packages/aws-cdk-lib/aws-stepfunctions/lib/states/distributed-map/item-reader.ts

```diff
@@ -12,6 +12,11 @@ export interface IItemReader {
    */
   readonly bucket: IBucket;
```

Comment:
should we change this attribute to be optional?

Reply:
No - because as per [Making properties optional](https://github.com/aws/jsii/blob/main/packages/jsii-diff/BREAKING_CHANGES.md#making-properties-optional):
> You are allowed to make inputs optional, but you are not allowed to make outputs optional.

Hence, we can only make properties of "props" (input) optional but not the properties of "interface" (output).